wordpress-mobile/WordPress-iOS
# Today widget localization

Fixes #15157

**Note**:
To achieve this, I copied the existing Today Widget `Localizable.strings`. Since this did not contain the "Today" string, I retrieved this from the main app, but since it was a manual process, translations will need to be verified. Also, we need to add translations of the placeholder/unconfigured view, that are not present at the moment.

To test:

- Install Today Widget, medium size (small size would not contain all the strings)
- make sure the strings look correct in the current device language
- change the device language (as many times as you want) and make sure that the strings still look correct

PR submission checklist:
